marray/__init__.py

```diff
@@ -79,20 +79,26 @@ def __setitem__(self, key, other):
             self.mask[key] = getattr(other, 'mask', False)
             return self.data.__setitem__(key, getattr(other, 'data', other))
 
+        def _data_mask_string(self, fun):
+            data_str = fun(self.data) + ", "
+            mask_str = fun(self.mask)
+            if len(data_str) + len(mask_str) <= 66:
+                join_char = ""
+                components = ["MaskedArray(", data_str, mask_str, ")"]
+            else:
+                join_char = "\n"
+                components = ["MaskedArray(",
+                              textwrap.indent(data_str, " " * 4),
+                              textwrap.indent(mask_str, " " * 4),
```

Comment:
I would have probably gone with 2 spaces since we don't need to follow the same syntax as python

Reply:
Why not follow the Python convention, though, especially if we do want it to be possible to recreate the array from the output?